src/Compilers/CSharp/Portable/Symbols/MemberSymbolExtensions.cs

```diff
@@ -453,48 +453,6 @@ internal static ImmutableArray<Symbol> GetExplicitInterfaceImplementations(this
             }
         }
 
-        internal static TypeSymbol GetTypeOrReturnType(this Symbol member)
```

Comment:
>internal static TypeSymbol GetTypeOrReturnType(this Symbol member) [](start = 8, length = 66)

Could you please share the rational behind moving these helpers to SymbolExtensions.cs?